### PR TITLE
fix(PlayCommand): Timeout Keeps running

### PR DESCRIPTION
**Pull request by lxndr-rl**
When a song ends and you play another song the timeout keeps running:

PlayError

- [x] This was fixed just clearing the timeout

### CODE REVIEW

**Comment by mzrtamp** [comment]
It doesn't happen to me...

File: src/commands/PlayCommand.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ export class PlayCommand extends BaseCommand {
                 createEmbed("error", `Invalid usage, use **\`${this.client.config.prefix}help play\`** for more information`)
             );
         }
+
```
Comment (mzrtamp) [inline src/commands/PlayCommand.ts]:
```suggestion
```

**Review by mzrtamp — CHANGES_REQUESTED** [review]

**Review by mzrtamp — APPROVED** [review]
Great

**Comment by lxndr-rl** [comment]
> It doesn't happen to me...

Set the time to remove a queue (`CONFIG_DELETE_QUEUE_TIMEOUT`) to a low number (example: 5) -> play something short (so you don't wait too long)
When finished, play something and the bot disconnect

**Comment by mzrtamp** [comment]
> > It doesn't happen to me...
> 
> Set the time to remove a queue (`CONFIG_DELETE_QUEUE_TIMEOUT`) to a low number (example: 5) -> play something short (so you don't wait too long)
> When finished, play something and the bot disconnect

Ooohhh...

**Pull request merged**